src/lib/types/index.ts

```diff
@@ -17,6 +17,9 @@ export interface SystemStats {
   memory_used: number;
   memory_free: number;
   memory_cached: number;
+  swap_memory: number;
```

Comment:
@laciferin2024 shouldn't this be called swap_space? like the other ones ?

Reply:
disk_space is correct. But swap is a virtual memory. I honestly wanted use `memory_swap` but `memory_swap_free` is too long .

Reply:
isn't that a mismatch between the TS interface attribute name and the Rust struct

Reply:
Yeah my bad. I don't see it the merge anymore in the main branch. What hapenned?

Reply:
I would suggest sticking with `swap_memory`

Reply:
We would definetely need to show swap for linux machines, its a big blow as htop does display it

Reply:
I thought you don't want to fix that typo

Reply:
why would i don't want to .

Reply:
Its done, i have made it `swap_memory`